src/api/v3/index.ts

```diff
@@ -0,0 +1,43 @@
+import express, { NextFunction, Request, Response } from 'express';
+import config from '../../config';
+import CacheControlHandler from '../lib/cacheControlHandler';
+import ConsumerKeyHandler from '../../auth/consumerKeyHandler';
+import { GraphQLErrorHandler } from '../error/graphQLErrorHandler';
+import { handleQueryParameters } from './inputs';
+import { BFFFxError } from '../../bfffxError';
+import Recommendations from '../../graphql-proxy/recommendations/recommendations';
+import { forwardHeadersMiddleware } from '../../graphql-proxy/lib/client';
+import { RecommendationsQueryVariables } from '../../generated/graphql/types';
+import { GlobalRecsResponse, responseTransformer } from './response';
+
+const router = express.Router();
+
+router.get(
+  '/firefox/global-recs',
+  // request must include a consumer
+  ConsumerKeyHandler,
+  CacheControlHandler('public, max-age=1800', config),
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const variables = handleQueryParameters(req.query);
+
+      if (variables instanceof BFFFxError) {
+        return next(variables);
+      }
+
+      const graphRes = await Recommendations({
```

Comment:
For future enhancement it: Given all requests in this repo is probably forwarding a request to the graph it would be great to have either a middleware or some factory to generate a graph client for us with all of the authentication parameters already handled.